src/demo-app/DemoApp.tsx

```diff
@@ -5,6 +5,10 @@ import SilxProvider from '../h5web/providers/silx/SilxProvider';
 import MockProvider from '../h5web/providers/mock/MockProvider';
 import HsdsProvider from '../h5web/providers/hsds/HsdsProvider';
 
+const HSDS_USERNAME = process.env.REACT_APP_HSDS_USERNAME || '';
+const HSDS_PASSWORD = process.env.REACT_APP_HSDS_PASSWORD || '';
+const HSDS_FILEPATH = process.env.REACT_APP_HSDS_FILEPATH || '';
```

Comment:
Since `.env` will always have empty values for these vars, they should already default to empty strings here.

Reply:
Actually, TS gives the type of `process.env.REACT_APP_HSDS_USERNAME` as `string | undefined`...